package-lock.json

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
I don't think that we want to add a global package.json/package-lock.json and you should not need to touch varfish/vueapp/*

Reply:
agreed